app/assets/stylesheets/frontpage.css.scss

```diff
@@ -28,25 +26,45 @@
   }
   margin-left: 2%;
   opacity: .94;
-  float: left;
 }
 
 .page-content
 {
-  width: 90%;
-  margin: 5% auto;
+  padding-top: 1%;
+  background-color: #cccccc;
+  width: 95%;
+  margin: 0 auto;
+  margin-bottom: 2%;
+}
+
+.all-shows-background {
+  padding-top: 2%;
+  border-top: 2px solid black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.